source/client-side-encryption/tests/README.rst

```diff
@@ -2693,302 +2694,343 @@ Create a ClientEncryption object named ``clientEncryption`` with these options:
+Use ``clientEncryption.encrypt()`` to encrypt the value 6. Ensure the type matches that of the encrypted field. For example, if the encrypted field is ``encryptedLong`` encrypt a BSON int64 type, not a BSON int32 type.
 
-Store the result in ``insertPayload``.
-
-Encrypt with the matching ``RangeOpts`` listed in `Test Setup: RangeOpts`_ and these ``EncryptOpts``:
+Encrypt using the following ``EncryptOpts``:
 
 .. code:: typescript
 
    class EncryptOpts {
-      keyId : <key1ID>
+      keyId : <key1ID>,
       algorithm: "RangePreview",
-      contentionFactor: 0
+      contentionFactor: 0,
+      rangeOpts: <RangeOpts for Type>,
    }
 
-Use ``clientEncryption`` to decrypt ``insertPayload``. Assert the returned value equals 6.
+Store the result in ``insertPayload``.
+
+Use ``clientEncryption`` to decrypt ``insertPayload``. Assert the returned value equals 6 and has the expected type.
 
 .. note::
 
    The type returned by ``clientEncryption.decrypt()`` may differ from the input type to ``clientEncryption.encrypt()`` depending on how the driver unmarshals BSON numerics to language native types.
-   Example: a driver may unmarshal a BSON long to a numeric type that does not distinguish between int64 and int32.
+   Example: a driver may unmarshal a BSON int64 to a numeric type that does not distinguish between int64 and int32.
 
-Case 2: can find encrypted range and return the maximum 
+
+Case 2: can find encrypted range and return the maximum
 ```````````````````````````````````````````````````````
 Use ``clientEncryption.encryptExpression()`` to encrypt this query:
 
 .. code:: javascript
 
-   //convert 6 and 200 to match the type of the encrypted field.
-   {"$and": [{"encrypted<Type>": {"$gte": 6}}, {"encrypted<Type>": {"$lte": 200}}]}
+   // Convert 6 and 200 to the encrypted field type
+   { "$and": [ { "encrypted<Type>": { "$gte": 6 } }, { "encrypted<Type>": { "$lte": 200 } } ] }
 
-Use the matching ``RangeOpts`` listed in `Test Setup: RangeOpts`_ and these ``EncryptOpts`` to encrypt the query:
+Encrypt using the following ``EncryptOpts``:
 
 .. code:: typescript
 
    class EncryptOpts {
-      keyId : <key1ID>
+      keyId : <key1ID>,
       algorithm: "RangePreview",
       queryType: "rangePreview",
-      contentionFactor: 0
+      contentionFactor: 0,
+      rangeOpts: <RangeOpts for Type>,
    }
 
 Store the result in ``findPayload``.
 
 Use ``encryptedClient`` to run a "find" operation on the ``db.explicit_encryption`` collection with the filter ``findPayload`` and sort the results by ``_id``.
 
-Assert these three documents ``{ "encrypted<Type>": 6 }, { "encrypted<Type>": 30 }, { "encrypted<Type>": 200}`` are returned.
+Assert the following three documents are returned:
+
+.. code:: javascript
 
+   // Convert 6, 30, and 200 to the encrypted field type
+   { "_id": 1, "encrypted<Type>": 6 }
+   { "_id": 2, "encrypted<Type>": 30 }
+   { "_id": 3, "encrypted<Type>": 200 }
```

Comment:
The original documents did not include `_id` fields, but I think that's helpful for matching. Breaking this out to a code block also made it more readable and easier to remind drivers to convert values.